### ISSUE TITLE
Constructing OSM Newsletter 2

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSM_To_Do_List#566]
About a month ago we sent out [the first OSM email Newsletter](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/547). It'd be good to assemble the next one, to highlight what is currently going on.

**Suggestions**:

1) Ongoing Pfizer story - David's synthesis (OpenSourceMalaria/Series4#36)
2) New Griffith targets - the carbamates (OpenSourceMalaria/Series4#43)
3) Marat's discovery of unexpected products (OpenSourceMalaria/Series4#46)
4) Results from Dundee (expected any day now, but we don't have them yet: OpenSourceMalaria/Series4#39)
5) hERG results from Ed's visit to Victor Chang (again, currently waiting on these: OpenSourceMalaria/Series4#37)
6) Ongoing Series 4 paper - [updating of the wiki](https://github.com/OpenSourceMalaria/OSMSeries4Paper1/issues), and what writing is needed.

Any other things we should highlight?

@mcoster - you had suggested an alternative way of doing newsletters. Did you want to pursue that now, or leave it till the next one? We're lacking a good way (permanent URL, for example) for people to sign up if they want to, but I think that ought to be on the landing page at opensourcemalaria.org

Anyone - I'm hoping we can assemble the newsletter via a shared account. If anyone behind the above suggestions, or anyone else, wants to write a few words with a picture, let me know and I can give you the details.

An issue related to the newsletters: GDPR is live from May 25th (tomorrow). Started a separate issue (OpenSourceMalaria/TechOps#2) about this, asking for advice. Hoping don't need to do anything. Probably do.
